scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -176,6 +177,11 @@
 	/** Realtime nanoseconds of last sending a flight */
 	private long flightSendNanos;
 
+	/** Expire realtime nanoseconds */
+	private long expiresNanos;
```

Comment:
Any idea about a naming which express that `expiresNanos` is a timestamp and `expiresNanosTimeout` a timeout duration ?

Reply:
Isn't "timeout" used for durations so far?
And I'm not sure, if we name the timestamps special, but I maybe wrong.

Reply:
I think you're right we do nothing special until now.
It's just I didn't get the meaning of this variable at first sight and I tell myself that maybe naming could help.
(not a strong opinion)

Maybe javadoc is enough ?